src/components/Identicon/index.tsx

```diff
@@ -1,4 +1,6 @@
 import * as React from 'react'
```

Comment:
do you think we can move this component to components repo? should be super straightforward

Reply:
good idea, but let's do this as a separate ticket, the amount of work have been put in this PR is already big enough